lib/launch_core/dev/sensor.c

```diff
@@ -56,3 +56,19 @@ int l_get_sensor_data(const struct device *const dev, int num_channels, enum sen
 
     return 0;
 }
+
+int l_get_sensor_data_float(const struct device *const dev, int num_channels, enum sensor_channel const *channels,
```

Comment:
any reason this isnt void if its always returning 0?

Reply:
In case there's a need in the near future. If there isn't by then, we can update.